lib/charms/alertmanager_k8s/v0/alertmanager_remote_configuration.py

```diff
@@ -0,0 +1,1890 @@
+                    "type": "string",
+                },
+                "message": {
+                    "description": "API request data as defined by the WeChat API.",
+                    "$ref": "#/definitions/string_or_template_ref_value",
+                    "default": '{{ template "wechat.default.message" . }}',
+                },
+                "message_type": {
+                    "description": "Type of the message type, supported values are `text` and `markdown`.",  # noqa: E501
+                    "type": "string",
+                    "enum": ["text", "markdown"],
+                    "default": "text",
+                },
+                "agent_id": {
+                    "$ref": "#/definitions/string_or_template_ref_value",
+                    "default": '{{ template "wechat.default.agent_id" . }}',
+                },
+                "to_user": {
+                    "$ref": "#/definitions/string_or_template_ref_value",
+                    "default": '{{ template "wechat.default.to_user" . }}',
+                },
+                "to_party": {
+                    "$ref": "#/definitions/string_or_template_ref_value",
+                    "default": '{{ template "wechat.default.to_party" . }}',
+                },
+                "to_tag": {
+                    "$ref": "#/definitions/string_or_template_ref_value",
+                    "default": '{{ template "wechat.default.to_tag" . }}',
+                },
+            },
+            "additionalProperties": False,
+        },
+        "weekdays": {
+            "description": "Days should be specified by name (e.g. 'Sunday'). For convenience, ranges are also accepted of the form : and are inclusive on both ends. Example: 'tuesday:Saturday' or 'Monday'.",  # noqa: E501
+            "type": "string",
+            "pattern": "^([Mm]onday|[Tt]uesday|[Ww]ednesday|[Tt]hursday|[Ff]riday|[Ss]aturday|[Ss]unday)(:($1))?$",  # noqa: E501
+        },
+        "years": {
+            "description": "A numerical value of years. Ranges are accepted. Ranges are inclusive on both ends. Example: '2020:2022' or '2030'.",  # noqa: E501
+            "type": "string",
+            "pattern": "^(2\\d\\d\\d)(:[2-9]\\d\\d\\d)?$",
+        },
+    },
+}
+DEFAULT_RELATION_NAME = "remote-configuration"
+DEFAULT_ALERTMANAGER_CONFIG_FILE_PATH = "/etc/alertmanager/alertmanager.yml"
+
+
+def load_config_file(path: str) -> dict:
+    """Reads given Alertmanager configuration file and turns it into a dictionary.
+
+    Args:
+        path: Path to the Alertmanager configuration file
+
+    Returns:
+        dict: Alertmanager configuration file in a form of a dictionary
+    """
+    if os.path.exists(path):
+        with open(path, "r") as config_yaml:
+            config = yaml.safe_load(config_yaml)
+        return config
+    else:
+        error_msg = "Given Alertmanager config file {} doesn't exist!".format(path)
```

Comment:
Better with f-strings?
```suggestion
        error_msg = f"Given Alertmanager config file {path} doesn't exist!"
```

Reply:
No, we can't use f-strings  in libs or we will loose compatibility with Python 3.5 (I.e. a bunch of machine charms)

Reply:
Exactly :D I used f-strings originally ;)

Reply:
Okay got it. Thanks for clarifying 😅